### PR TITLE
Bug fixes and features addition. [BEBA advanced]

### PR DESCRIPTION
**Pull request by angelotulumello**

### CODE REVIEW

File: oflib-exp/ofl-exp-beba.c [hunk 1]
```diff
@@ -108,6 +108,7 @@ ofl_structs_extraction_unpack(struct ofp_exp_set_extractor const *src, size_t *l
         }
         dst->table_id = src->table_id;
         dst->field_count=ntohl(src->field_count);
+        dst->biflow = src->biflow;  //da surce a dst
```
Comment (DavideSanvito) [inline oflib-exp/ofl-exp-beba.c]: Add a more meaningful comment, in Italian

File: udatapath/packet_handle_std.c [hunk 2]
```diff
@@ -451,7 +451,7 @@ packet_handle_std_validate(struct packet_handle_std *handle) {
     int i = 0;
     uint32_t current_global_state = OFP_GLOBAL_STATE_DEFAULT;
     gettimeofday(&tv,NULL);
-    timestamp = (1000000 * tv.tv_sec + tv.tv_usec)/1000; // timestamp in ms
```
Comment (DavideSanvito) [inline udatapath/packet_handle_std.c]: Why did you change the granularity?

Comment (angelotulumello) [inline udatapath/packet_handle_std.c]: For the EWMA calculation an ms granularity was too small. Now, in my use case, I'm doing an average before the EWMA filter so I guess that the ms could be sufficient. Therefore, it wouldn't be a problem for me to get back to milliseconds, if that matters.
